src/novel_applications/payment/private_payment.md

```diff
@@ -0,0 +1,167 @@
+
+## Additional thoughts and questions
+
+#### A better UX on Miden: Account abstraction
+
+*Question: Why would a user have a better UX on Miden than on another blockchain?*
+
+Ethereum has set the standard for accounts in the VM-enabled blockchain world. There are two types of accounts in Ethereum:
+
+- Externally owned accounts (EOA): are created by generating a public/private ECDSA key pair. Does not hold code. Can initiate transactions.
+- Contract accounts (Smart contracts): are created by being deployed on the Ethereum blockchain by an EOA. Holds code which can be executed. Can't initiate transactions.
+
+We clearly understand here that Ethereum has made the choice to separate user accounts from executable code. What if we could merge both to enable programmable user accounts? Welcoming `Account abstraction`, which can be defined as follows:
+
+"Account abstraction is a method of setting up a blockchain network in which users' assets are stored exclusively in smart contracts, and not in external accounts (External Owned Accounts, EOAs). When using this approach, a crypto wallet turns into a unique smart contract that can be programmed for various purposes."
+
+[What is account abstraction and why is it important - Medium - Aleksander](https://medium.com/@alex-100/what-is-account-abstraction-and-why-is-it-important-9627a4ced4f3)
+
+Miden supports full account abstraction, enabling full programmability of user accounts defined as follows:
+
+<p align="center">
+    <img src="../../assets/images/account.png" width="40%" alt="Account definition">
+</p>
+
+The `Miden VM code` field hints that all Miden accounts are `abstracted`, which enables arbitrary logic to be executed against them, opening the door to unbounded functionalities. Using these innovations, we can imagine improvements in UX like social recovery, abstracted seed phrases, Face-ID signing, and many more.
+
+#### Making private transfers
+
+*Question: How can the application developer use Miden to make private transfers?*
+
+Using the [Miden SDK](https://www.npmjs.com/package/@demox-labs/miden-sdk?activeTab=readme) or the [Miden client](https://github.com/0xPolygonMiden/miden-client), an application developer can import Miden core components into their application, enabling them to create accounts, use the [Miden VM](https://github.com/0xPolygonMiden/miden-vm) to make state transitions (transactions) and generate notes, [Miden prover](https://github.com/facebook/winterfell) to make proofs and preserve privacy, and more.
+
+#### Account storage
+
+*Question: Where and how would I store my account?*
+
+The user account, assets and private key would be stored securely on device. For additional redundancy, we could add social recovery solutions, storage of an encrypted account state in the cloud, and more.
+
+#### Secure enclave and Keystore
```

Comment:
nit: keystore (lowercase)

Reply:
I think that we should keep it Upercased because `Keystore` is the name of the secure element of Android devices:
- Apple -> `Secure enclave`
- Android -> `Keystore`